feature-libs/order/components/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -46,7 +46,13 @@ <h3>
         <table
           id="replenishment-order-history-table"
           class="table cx-replenishment-order-history-table"
+          role="table"
         >
+          <caption *cxFeatureLevel="'5.1'" class="cx-visually-hidden">
```

Comment:
Please document elements affected by `cxFeatureLevel` in `6_0.md` (you may need to make a new file like `5_0.md`).